src/server/server.py

```diff
@@ -1,333 +1,48 @@
-        self.msg = '{"success": %(success)s, "message":"%(message)s"}'
-
-    def run(self):
-        """
-        Start udp server
-        """
-        self.sock = socket.socket(socket.AF_INET,
-                                  socket.SOCK_DGRAM)
-        self.sock.bind(("0.0.0.0", self.udp_port))
-        self.sock.setblocking(0)
-        self.sock.settimeout(5)
-        while self.is_listening:
-            try:
-                data, address = self.sock.recvfrom(1024)
-            except socket.timeout:
-                continue
-
-            try:
-                data = json.loads(data)
-                try:
-                    identifier = data['identifier']
-                except KeyError:
-                    identifier = None
-
-                try:
-                    room_id = data['room_id']
-                except KeyError:
-                    room_id = None
-
-                try:
-                    payload = data['payload']
-                except KeyError:
-                    payload = None
-
-                try:
-                    action = data['action']
-                except KeyError:
-                    action = None
-
-                try:
-                    if room_id not in self.rooms.rooms.keys():
-                        raise RoomNotFound
-                    self.lock.acquire()
-                    try:
-                        if action == "send":
-                            try:
-                                self.rooms.send(identifier,
-                                                room_id,
-                                                payload['message'],
-                                                self.sock)
-                            except BaseException:
-                                pass
-                        elif action == "sendto":
-                            try:
-                                self.rooms.sendto(identifier,
-                                                  room_id,
-                                                  payload['recipients'],
-                                                  payload['message'],
-                                                  self.sock)
-                            except BaseException:
-                                pass
-                    finally:
-                        self.lock.release()
-                except RoomNotFound:
-                    print("Room not found")
-
-            except KeyError:
-                print("Json from %s:%s is not valid" % address)
-            except ValueError:
-                print("Message from %s:%s is not valid json string" % address)
 
-        self.stop()
+    # Get the Port and Debug mode from command line options
+    options.parse_command_line()
 
-    def stop(self):
-        """
-        Stop server
-        """
-        self.sock.close()
+    # create logger for app
+    logger = logging.getLogger('app')
+    logger.setLevel(logging.INFO)
```

Comment:
There is a logging_config already in place under logging_config.ini so you don't need to set this up here. You can just call logger = logging.getLogger().